thrust/thrust/functional.h

```diff
@@ -927,7 +927,7 @@ THRUST_UNARY_FUNCTOR_VOID_SPECIALIZATION(identity, THRUST_FWD(x));
  *  \see binary_function
  */
 template <typename T = void>
-struct maximum
+struct maximum : cuda::std::maximum<T>
```

Comment:
That should be `cuda::maximum`, not `cuda::std::maximum`.

Reply:
Whoops, fixed